recipes/blend2d/meta.yaml

```diff
@@ -0,0 +1,61 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+{% set name = "blend2d" %}
+{% set version = "beta12" %}
```

Comment:
Can you pick a better version number than this? I fear that this might conflict with releases later on.

Reply:
It is not my package, I am just doing a recipe for it.